modules/userinterfacegl/glsl/renderui.frag

```diff
@@ -33,7 +33,7 @@ uniform sampler2DArray arrayTexSampler;
 // texture indices for normal, pressed, checked, 
 // halo normal, halo pressed, halo checked
 // border normal, border pressed, border checked
-uniform int arrayTexMap[9] = { 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+uniform int arrayTexMap[9] = int[]( 0, 0, 0, 0, 0, 0, 0, 0, 0 );
```

Comment:
this one is quite interesting. I think we need the `{}` initialization in other shaders, too...

Reply:
Did a quick check in the plottinggl module but did not find any other. Could exist elsewhere though